modules/mysql/mysql_test.go

```diff
@@ -34,6 +35,8 @@ func TestMySQL(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	mustConnectionString := container.MustConnectionString(ctx,"tls=skip-verify")
+	require.Equal(t,connectionString,mustConnectionString)
```

Comment:
Super NIT: all the assertions in this module are done without testify.

I have to think about if using testify everywhere, or remove it everywhere, but not there yet. I'd keep this file at it was, without testify 🙏

Reply:
Will do.